### PR TITLE
Feat external proponent landing page

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH CARD 1411](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1411)

### CODE REVIEW

File: app/pages/cif-external/index.tsx [hunk 1]
```diff
@@ -82,6 +83,9 @@ export function ExternalProjects({
     [allProjectStatuses.edges]
   );
 
+  // Growthbook - external-operators
+  if (!useShowGrowthbookFeature("external-operators")) return null;
+
```
Comment (BCerki) [inline app/pages/cif-external/index.tsx:88]: Thanks, I forgot to add this!

File: app/components/Layout/SubHeader.tsx [hunk 2]
```diff
@@ -30,17 +30,17 @@ export default function SubHeader(props: Props) {
           <Link
             href={
               // The external user subheader doesn't contain an operators link, so we can use its presence/absence to check if we're routing internal vs. external users
```
Comment (BCerki) [inline app/components/Layout/SubHeader.tsx:32]:
```suggestion
         
```

File: app/cypress/integration/cif-external/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+describe("the external projects page", () => {
```
Comment (BCerki) [inline app/cypress/integration/cif-external/index.spec.js:1]: Something to discuss with the whole team--how much e2e testing do we want to do at this stage? A lot of the first slice we already know will be temporary, so might be worth keeping the e2e testing light

**Review by BCerki — APPROVED** [review]

File: app/pages/cif-external/index.tsx (hunk 1 above)
Comment (Sepehr-Sobhani) [reply app/pages/cif-external/index.tsx:88]: Thanks for creating the feature flag :)

File: app/cypress/integration/cif-external/index.spec.js (hunk 3 above)
Comment (Sepehr-Sobhani) [reply app/cypress/integration/cif-external/index.spec.js:1]: Good point! Let me share this in the devs channel.

**Review by BCerki — APPROVED** [review]

**Review by BCerki — APPROVED** [review]

**Review by BCerki — APPROVED** [review]

**Pull request merged**